docs/cloud-adoption/operations/operational-fitness-review.md

```diff
@@ -0,0 +1,127 @@
+
+An output from these activities is a set of **scorecard metrics** for service operations. The metrics are categorized in terms of non-functional design criteria, i.e. the quality pillars, such as availability, scalability, disaster recovery, and so on. Scorecard metrics are an expression of the desired criteria that the service is expected to operationally meet. These metrics can be expressed at any level of granularity that is appropriate for the service operation.
+
+The scorecard should be expressed in simple terms to facilitate meaningful discussion between the business owners and engineering. For example, a scalability scorecard metric could be expressed as _green_ for performing at the desired criteria, _yellow_ for failing to meet the desired criteria but actively implementing a planned remediation, and _red_ for failing to meet the desired criteria with no plan or action.
+
+It is important to emphasize that these metrics should directly reflect business needs.
+
+### Service Review Phase
+
+The service review phase is core of the operational fitness review process.
+
+- **Measure Service Metrics** Using the scorecard metrics, the services should be monitored to ensure that they meet the business expectations. This means that service monitoring is essential. If you are not able to monitor a set of services with respect to the non-functional requirements, then the corresponding scorecard metrics should be considered red. In this case, the first step for remediation is to implement the appropriate service monitoring.
+For example, if the business expects a service operation to operate with 99.99% availability but there is no production telemetry in place regarding the availability of the service, then it should be assumed that you are not meeting the requirement.
+
+- **Plan Remediation** For each service operation with metrics that fall below an acceptable threshold, determine the cost of remediating the service to bring operation to an acceptable metric. If the cost of remediating the service is greater than the expected revenue generation of the service, move on to consider the non-tangible costs such as customer experience. For example, if customers have difficulty placing a successful order using the service, they may choose a competitor instead.
+
+- **Implement Remediation** After the business owners and engineering converge on a plan, it should be implemented. The status of the implementation should be reported whenever scorecard metrics are reviewed.
+
+This process is iterative, and ideally your enterprise should have a team dedicated to owning it. This team should meet regularly to review existing remediation projects, kick off the fundamentals review of new workloads, and track the enterprise's overall scorecard. The team should have the authority to ensure accountability for remediation teams that are behind schedule or fail to meet metrics.
```

Comment:
First use of "fundamentals review" in this doc. Same as operational fitness review?